start-workspace-from-url.adoc

```diff
@@ -0,0 +1,88 @@
+:prod-short: Che
+
+= Starting a workspace from an URL
+
+After using the getting started examples from the Dashboard, users will want to create workspaces from existing Git repositories.
+Create a workspace from a Git repository by entering a custom URL on the {prod-short} dashboard page.
+
+== Short URL syntax
+
+In the earlier versions of {prod-short}, the {prod-short} URL syntax began with `/f/?url=`, for example `https://my-che-server/f/?url=https://github.com/my/repository`, but it was complex and error prone. While that syntax is still supported, short syntax should be used.
+
+Start a workspace with a GitHub, GitLab, or Bitbucket repository by using the {prod-short} URL prepended with a `#` character and appended with the repository link. Using the example of Eclipse Che hosted by Red Hat: link:https://workspaces.openshift.com#https://github.com/eclipse-che/che-server[].
+
+{prod-short} starts a workspace and clones the repository `https://github.com/eclipse-che/che-server`.
+
+{prod-short} will search for a `.devfile.yaml` or `devfile.yaml` file in the given repository. The devfile describe the containers and commands to use for that project. Omitting that file will result in a default container. The default container provides different languages (including Java, Node.js, Python, PHP, Golang, etc) and tools (including `curl`, `jq`, `yq`, etc).
+
+Devfile information: https://devfile.io/[devfile specification]
```

Comment:
```suggestion
= Starting a workspace by entering a URL in your internet browser

After using the getting started examples from the Dashboard, users will want to create workspaces containing workspace clones of existing Git repositories.
Create a workspace with a workspace clone of a Git repository by entering a custom URL on the {prod-short} dashboard page.

== Short URL syntax for cloning hosted git repositories

In the earlier versions of {prod-short}, the {prod-short} URL syntax began with `/f/?url=`, for example `https://my-che-server/f/?url=https://github.com/my/repository`, but it was complex and error prone. While that syntax is still supported, short syntax should be used as follows:

To start a workspace with a workspace clone of a GitHub, GitLab, or Bitbucket repository, go to your browser and enter the {prod-short} URL prepended with a `#` character and appended with the repository link. Using the example of Eclipse Che hosted by Red Hat: link:https://workspaces.openshift.com#https://github.com/eclipse-che/che-server[].

For GitHub:

* To clone the default branch on GitHub to the starting workspace: `#https://github.com/_<user_or_org>_/_<repository>_`
+
Example: `#https://github.com/eclipse-che/che-plugin-registry`

* To clone a specified branch on GitHub to the starting workspace: `#https://github.com/_<user_or_org>_/_<repository>_/tree/_<branch-name>_`
+
Example: `#https://github.com/eclipse-che/che-plugin-registry/tree/7.39.x`

* To clone the branch of a pull request on GitHub to the starting workspace: `#https://github.com/_<user_or_org>_/_<repository>_/pull/_<pull-request-id>_`
+
Example: `#https://github.com/eclipse-che/che-plugin-registry/pull/1097`

For GitLab:

* To clone the default branch on GitLab to the starting workspace: `#https://gitlab.com/_<user_or_org>_/_<repository>_`

* To clone a specified branch on GitLab to the starting workspace: `#https://gitlab.com/_<user_or_org>_/_<repository>_/-/tree/_<branch>_`

////
Examples of GitLab URLs:
https://gitlab.com/doctormo
https://gitlab.com/nathanal/inkscape/-/tree/inkscape1902
https://gitlab.com/inkscape/inkscape/-/merge_requests/3705
////

{prod-short} then starts a workspace and clones the repository `\https://github.com/eclipse-che/che-server`.

{prod-short} the searches for a `.devfile.yaml` or `devfile.yaml` file in the given repository. The devfile describes the containers and commands to use for that project. Omitting this file will result in a default container. The default container provides different languages (including Java, Node.js, Python, PHP, Golang, etc) and tools (including `curl`, `jq`, `yq`, etc).

Devfile information: https://devfile.io/[devfile specification]
```

Reply:
My suggestion for `= Starting a workspace by entering a URL in your internet browser` is most likely wrong in the text, but can a user just paste a URL into their browser (without going through the dashboard)?

Reply:
yes you don't need to open dashboard first if you know where che is deployed

for example I can type in my browser https://workspaces.openshift.com#https://github.com/eclipse/che directly